pkg/object/object.go

```diff
@@ -45,9 +48,34 @@ type OpenCompose struct {
 	Volumes  []Volume
 }
 
+func validateReplicas(replicas *int32) error {
+	if replicas != nil && *replicas < 0 {
+		return fmt.Errorf("Replica count is negative")
+	}
+	return nil
+}
+
+func (s *Service) Validate() error {
+	// validate service name, like it cannot have underscores, etc.
+
+	// validate containers
+
+	// validate replicas
+	if err := validateReplicas(s.Replicas); err != nil {
```

Comment:
than you separately testing one `if` statement? 
I don't know, but for me it looks like it is just overcomplicating things

`Validate` can be tested as whole.